.circleci/config.yml

```diff
@@ -10,8 +10,23 @@ jobs:
       JVM_OPTS: -Xmx3200m
     steps:
       - checkout
+      - run:
+          name: Run Ktlint Checks
+          command: |
+            curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.39.0/ktlint
+            chmod a+x ktlint
+            ./ktlint
```

Comment:
Doesn't detekt have ktlint integration? I wouldn't expect to have to download it separately.